services/telemetry/telemetry.go

```diff
@@ -153,20 +200,44 @@ func (s *Service) sendOnce(ctx context.Context) error {
 		return err
 	}
 
-	payload := s.makePayload(settings.Telemetry.UUID)
-	return s.sendRequest(ctx, payload)
+	var wg errgroup.Group
+
+	wg.Go(func() error {
+		payload := s.makeV1Payload(settings.Telemetry.UUID)
```

Comment:
Why can't we move calling makeV1Payload inside sendV1Request?

Reply:
It's just decomposition. It's easier to maintain and test isolated chunks of functionality.

Reply:
More specifically, v1 is very picky about input format internally, so we have to test a function returning request body separately.

Reply:
that's why I said `calling`, I don't see any test where we call `sendV1Request` without calling `makeV1Payload`.
Having `makeV1Payload` as a separate method makes sense.
But, yes, let it keep as is to have similar logic with V2